src/xdocs/checks/sizes/filelength.xml

```diff
@@ -35,7 +35,7 @@
               <td>Specify the file type extension of files to process.</td>
               <td><a href="../../property_types.html#String.5B.5D">String[]</a></td>
               <td><code>all files</code></td>
-              <td>5.0</td>
+              <td>8.24</td>
```

Comment:
We are not expecting a change in versions. Prove to me the original `5.0` is wrong.

Reply:
workaround-ed.
actual fix will be https://github.com/checkstyle/checkstyle/issues/14052

Reply:
If someone else is going to work on this, they we need to document with a comment that says "until ..."

Reply:
It is planned for most near days.
I split it separate to keep moving in migration quicker and not be stopping all until some issue is resolved.
As I found solution only recently, so trying to reuse same hack as we use to have now.

Reply:
link is added